File: app/src/test/java/com/mgaetan89/showsrage/fragment/SettingsAboutFragmentTest.java
package com.mgaetan89.showsrage.fragment;

import com.mgaetan89.showsrage.R;

import org.junit.Test;

import static org.assertj.core.api.Assertions.assertThat;

public class SettingsAboutFragmentTest {
	@Test
	public void getTitleResourceId() {
		assertThat(new SettingsAboutFragment().getTitleResourceId()).isEqualTo(R.string.about);
	}

	@Test
	public void getXmlResourceFile() {
		assertThat(new SettingsAboutFragment().getXmlResourceFile()).isEqualTo(R.xml.settings_about);
	}
}

File: app/src/test/kotlin/com/mgaetan89/showsrage/fragment/SettingsAboutFragmentTest.kt
package com.mgaetan89.showsrage.fragment

import com.mgaetan89.showsrage.R
import org.assertj.core.api.Assertions.assertThat
import org.junit.Test

class SettingsAboutFragmentTest {
	@Test
	fun getTitleResourceId() {
		assertThat(SettingsAboutFragment.newInstance().getTitleResourceId()).isEqualTo(R.string.about)
	}

	@Test
	fun getXmlResourceFile() {
		assertThat(SettingsAboutFragment.newInstance().getXmlResourceFile()).isEqualTo(R.xml.settings_about)
	}
}
